.github/workflows/daily-builds.yml

```
@@ -53,3 +53,9 @@ jobs:
     if: ${{ needs.check_for_commits.outputs.new_commit_count > 0 || github.event_name == 'workflow_dispatch' }}
     uses: ./.github/workflows/deb-daily-builds.yml
     secrets: inherit
+
+  snapcraft8-daily:
```

Maybe just name it checkbox-snapcraft8-daily for consistency? 
```suggestion
  checkbox-snapcraft8-daily:
```